src/app/global/collection-wrapper/CollectionRoutesWrapper.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { replace } from 'react-router-redux';
+
+import { updateWorkingOn } from '../../config/actions'
+
+import log from '../../utilities/logging/log'
+import collections from '../../utilities/api-clients/collections'
+import notifications from '../../utilities/notifications'
+
+const propTypes = {
+    params: PropTypes.shape({
+        collectionID: PropTypes.string.isRequired,
+    }),
+    dispatch: PropTypes.func.isRequired,
+    location: PropTypes.shape({
+        pathname: PropTypes.string.isRequired,
+    }),
+    rootPath: PropTypes.string.isRequired,
+    workingOn: PropTypes.shape(),
+    children: PropTypes.element
+};
+
+export class CollectionRoutesWrapper extends Component {
+
+    componentDidMount = () => {
```

Comment:
Shouldn't this be a verb; for the name of a function?

Reply:
Do you mean `componentDidMount`? `componentDidMount` is a React lifecycle method.

Reply:
My limited knowledge of React showing here :D